notebooks/TFoS.ipynb

```diff
@@ -25,64 +26,74 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "# CPU Config\n",
-    "conf = SparkConf().setAppName('Mnist') \\\n",
-    "                  .set('spark.mesos.executor.docker.image', 'mesosphere/mesosphere-data-toolkit:latest') \n"
+    "# Remove existing models/artifacts if any\n",
+    "subprocess.check_output('hdfs dfs -rm -R -f -skipTrash mnist', shell=True)\n",
+    "subprocess.check_output('hdfs dfs -rm -R -f -skipTrash mnist_model', shell=True)\n",
+    "subprocess.check_output('hdfs dfs -rm -R -f -skipTrash predictions', shell=True)\n",
+    "subprocess.check_output('rm -Rf mnist tensorflowonspark', shell=True)\n",
+    "subprocess.check_output('rm -f mnist.zip', shell=True)"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "# GPU Config\n",
-    "#conf = SparkConf().setAppName('Mnist-GPU') \\\n",
-    "#                  .set('spark.mesos.executor.docker.image', 'mesosphere/mesosphere-data-toolkit:latest-gpu') \\\n",
-    "#                  .set('spark.mesos.gpus.max', 5) \\\n",
-    "#                  .set('spark.mesos.executor.gpus', 1)"
+    "# Clone the repo with the adjusted TF 1.11 APIs in mnist_dist/mnist_spark\n",
```

Comment:
Do you believe this will be merged soon? It feels a bit weird/unstable to use such kind of custom branch.

Reply:
It just got merged, I will update this PR accordingly

Reply:
+1